### PR TITLE
V2 Main 

### LINKED ISSUE #3765: 2.0 Event Types / Single {View}
[View in Figma →](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1413%3A65854)

This is a big one with multiple tabs. It's likely we may create separate issues for this.

- [x]  [Event setup](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1413%3A66246)
- [ ] [Availability](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1422%3A65944)
- [x] [Limits](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1526%3A75772)
- [ ] [Advanced](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1422%3A65954)
- [ ] [Recurring](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1422%3A65964)
- [ ] [Apps](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1413%3A111438)
- [ ] [Workflows](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=13992%3A98804)

Image

### LINKED ISSUE #3766: 2.0 Bookings {View}
[View in Figma →](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1413%3A65855)

Image

### PR DESCRIPTION
**Pull request by sean-brydon**
V2. WIP

### CODE REVIEW

**Comment by leog** [comment]
@sean-brydon very good job so far!

JM2C, I think @PeerRich removed all shadows from everywhere to avoid those from now on and also @Jaibles asked to only use `rounded-sm` if I'm not mistaken, just to see if v2 should be aligned with those changes that were already applied in the app

**Comment by PeerRich** [comment]
@sean-brydon whats missing to go "ready for review"?

File: apps/web/components/v2/eventtype/EventAdvancedTab.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
+import { EventTypeSetupInfered, FormValues } from "pages/v2/event-types/[type]";
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/EventAdvancedTab.tsx:1]: These set of components in V2 need to stay in here - mainly for using inferred types making this a lot easier and more future proof.

File: apps/storybook/.storybook/preview.js [hunk 2]
```diff
@@ -22,3 +22,13 @@ export const parameters = {
     Provider: RouterContext.Provider,
   },
 };
+
+window.getEmbedNamespace = () => {
+  const url = new URL(document.URL);
+  const namespace = url.searchParams.get("embed");
+  return namespace;
+};
```
Comment (sean-brydon) [inline apps/storybook/.storybook/preview.js:30]: Resolve embed requirements in storybook

File: packages/tsconfig/nextjs.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "esModuleInterop": true,
     "module": "esnext",
     "resolveJsonModule": true,
-    "strict": true,
+    "strict": false,
```
Comment (zomars) [inline packages/tsconfig/nextjs.json:18]: This is your type check failure @sean-brydon

File: apps/web/tsconfig.json [hunk 4]
```diff
@@ -17,7 +17,8 @@
     "../../packages/types/*.d.ts",
     "../../packages/types/next-auth.d.ts",
     "**/*.ts",
-    "**/*.tsx"
+    "**/*.tsx",
+    "../../packages/ui/v2/core/TableActions.tsx"
```
Comment (zomars) [inline apps/web/tsconfig.json:21]:
```suggestion
```

File: apps/web/pages/v2/event-types/index.tsx [hunk 5]
```diff
@@ -0,0 +1,640 @@
...
+} from "@calcom/ui/v2/core/Dropdown";
+import Shell from "@calcom/ui/v2/core/Shell";
+import CreateEventTypeButton from "@calcom/ui/v2/modules/event-types/CreateEventType";
+
+import { withQuery } from "@lib/QueryCell";
+import classNames from "@lib/classNames";
+import { HttpError } from "@lib/core/http/error";
+
+import { EmbedButton, EmbedDialog } from "@components/Embed";
+import EventTypeDescription from "@components/eventtype/EventTypeDescription";
```
Comment (CarinaWolli) [inline apps/web/pages/v2/event-types/index.tsx:32]: We don't use the v2 component for EventTypeDescription here. @sean-brydon is that on purpose?

**Review by zomars — APPROVED** [review]
LGTM

**Comment by zomars** [comment]
Merging as we need to test this on staging ASAP

**Pull request merged**